runtime/nls/cfre/cfrerr.nls

```diff
@@ -1472,3 +1472,20 @@ J9NLS_CFR_LVTT_DUPLICATE.explanation=Please consult the Java Virtual Machine Spe
 J9NLS_CFR_LVTT_DUPLICATE.system_action=The JVM will throw a verification or classloading-related exception such as java.lang.ClassFormatError.
 J9NLS_CFR_LVTT_DUPLICATE.user_response=Contact the provider of the classfile for a corrected version.
 # END NON-TRANSLATABLE
+
+# ClassFile is not translatable
+# Record is not translatable
+J9NLS_CFR_ERR_TWO_RECORD_ATTRIBUTES=ClassFile has two record attributes
```

Comment:
Can you adapt this to be similar to the other "multiple" attribute messages?  ie: like `J9NLS_CFR_ERR_MULTIPLE_STACK_MAP_ATTRIBUTES`